tensorflow_federated/python/research/utils/checkpoint_manager.py

```diff
@@ -222,4 +223,4 @@ def _round_num(self, checkpoint_path: str) -> int:
   def _get_all_checkpoint_paths(self) -> List[str]:
     """Returns all the checkpoint paths managed by the instance."""
     pattern = os.path.join(self._root_dir, '{}*'.format(self._prefix))
-    return tf.io.gfile.glob(pattern)
+    return glob(pattern)
```

Comment:
It would be preferable r to continue to use `tf.io.gfile`, which has support for more filesystems (e.g. HDFS (hdfs://) and Google Cloud Storage (gcs://)) mentioned here https://github.com/tensorflow/tensorflow/blob/b36436b087bd8e8701ef51718179037cccdfc26e/tensorflow/python/platform/gfile.py#L49

Here, could we rather filter the list before returning by using [`tf.io.gfile.isdir`](https://www.tensorflow.org/api_docs/python/tf/io/gfile/isdir`)?

It would be extra helpful if we could start an issue on [github.com/tensorflow/tensorflow](http://www.github.com/tensorflow/tensorflow) about Windows behaving differently here, and leave a code comment linking that issue here as well.

Reply:
ok, will change that in a bit, using listdir and filter with self._prefix should give the exact previous behaviour

Reply:
@ZacharyGarrett done